src/components/ViewRequest/DetailSections/JOADetails.tsx

```diff
@@ -24,35 +14,15 @@ const JOADetails = ({
   const request = useRequest(Number(params.requestId));
 
   return (
-    <Card style={{ margin: "0.25em 0px" }}>
-      <Accordion collapsible>
-        <AccordionItem value="joa">
-          <CardHeader
-            header={
-              <AccordionHeader>
-                <Subtitle1>JOA</Subtitle1>
-              </AccordionHeader>
-            }
-            action={
-              <Button
-                appearance="transparent"
-                icon={<EditIcon />}
-                aria-label="Edit"
-                onClick={() => {
-                  setEditSection("JOA");
-                  setIsEditOpen(true);
-                }}
-              >
-                Edit
-              </Button>
-            }
-          />
-          <AccordionPanel>
-            {request.data && <ViewRequestJOADetails data={request.data} />}
-          </AccordionPanel>
-        </AccordionItem>
-      </Accordion>
-    </Card>
+    <DetailsTemplate
+      sectionName="JOA"
+      sectionDescription="JOA"
+      setEditSection={setEditSection}
+      setIsEditOpen={setIsEditOpen}
+      detailSelection="joaPostDate"
+    >
+      {request.data && <ViewRequestJOADetails data={request.data} />}
+    </DetailsTemplate>
```

Comment:
Just a thought to take this one step further.  These Details components are basically composed of 2 components and 2 useHooks.  Is there much value in keeping them as separate files?  Or just move them straight into Details.tsx where the hooks are already there, and is where the callback functions themselves live, so you don't have to pass them down twice.